thefuck/rules/adb_unknown_command.py

```diff
@@ -0,0 +1,28 @@
+from thefuck.utils import is_app, get_closest, replace_argument
+
+
+# the most common ADB commands
+_ADB_COMMANDS = [
```

Comment:
There are more, do we want them all?

Reply:
Yeah, I think we may as well go ahead and list them all.